### PR TITLE
New Doc - Defend WordPress or Drupal Sites Against DDoS Attacks with Sucuri

### PR DESCRIPTION
**Pull request by rachelwhitton**
Replaces #763

@atjuch - ran into some issues with the original PR, creating this one for easier edits. The first commit is your most up to date revision, so it's preserved incase my edits were not friendly. Can you review for accuracy? 

@nataliejeremy can you do a style/grammar revision? 

@bmackinney can you review?

### CODE REVIEW

**Comment by atjuch** [comment]
@rachelwhitton I feel that you cut out a lot of important information. This guide now is way too basic and doesn't describe what it does. I think there were too many screenshots removed and too much context copy removed to what it protects against.

I think we should use my latest version I wrote and trim that down a bit. I appreciate the simplicity and your efforts for sure! It just needs more information.

**Comment by rachelwhitton** [comment]
@atjuch I hear you - we can revert back to the previous state of the guide. It was unclear which services came with the sucuri plans - e.g. protection against brute force attacks or malware injection, does that come with the same plan as the cloud proxy service? The guides previous state mentioned more advantages in passing but didn't discuss how they were enabled/configured, so I refocused the guide on the cloud proxy service. If there are additional features of the service we want to showcase we should touch on how they are configured/accomplished. 

I'd like @bmackinney or @nataliejeremy to weigh in before we restore the previous version.

**Comment by nataliejeremy** [comment]
@atjuch @rachelwhitton +1 for removing most of the screenshots. It's best practice as their UI will change and the images will be outdated.

File: source/docs/guides/defend-wordpress-and-drupal-with-sucuri.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+![Sucuri Add New Domain](/source/docs/assets/images/sucuri-add-new-domain.png)
+
+CloudProxy will display a status bar which will configure the account settings for your domain name. It should not take more than 1-2 minutes and will close automatically when finished.
+
+## Configure DNS
+
+Copy the **Firewall IP address**, located in **General Settings**. Log in to your DNS provider and delete the www CNAME record pointing to `env-yoursite.pantheon.io`. Create an A record for www using the Firewall IP address provided by Sucuri.
+
+Return to **General Settings** in your Sucuri account. After your DNS changes are detected, you will be able to configure the domain's **Hosting IP Address**. While the name suggests otherwise, you can provide a FQDN as an entry. Enter the destination for your site, found in the Domains/SSL tab by clicking **Show recommended DNS records**.
```
Comment (bmackinney) [inline source/docs/guides/defend-wordpress-and-drupal-with-sucuri.md]: "Enter **all** destinations for your site..."

**Comment by bmackinney** [comment]
I'm in agreement with @rachelwhitton but I think we can cut this down to a very short article or even a paragraph in `/sites/domains.md`. If I understand the process correctly, this boils down to: 

### Prerequisites

 - Sucuri Account
 - Live Pantheon site

### Process

1. [Activate Sucuri Cloud Proxy](https://kb.sucuri.net/cloudproxy/Configuration/getting-started)
2. Follow the instructions they provide for DNS
3. Add All destinations for your site to the list of hosts

I can't find the issue this or the previous PR are closing. In the future, please do not write a guide or an article without first creating an issue. We can then decide the best course of action for resolving the issue, without unnecessary effort.

**Comment by rachelwhitton** [comment]
@bmackinney @nataliejeremy  +1 for making it a standalone article in the domains dir. The domains/DNS doc has quite a bit of info already but we can link it in there. 

@atjuch are there features outside of the proxy service we need to document/discuss?

**Comment by atjuch** [comment]
After reviewing the article again I think that @rachelwhitton has the best intentions and great idea! If we need to update it later on and users are saying it's hard to follow we can adjust as needed.

So I say +1 and let's get it out!

**Comment by rachelwhitton** [comment]
@bmackinney can you do another review? Felt like the DNS config section warranted it's own doc. I added a question in the FAQ of the domains doc that links to this short doc.

File: source/docs/articles/sites/domains/defend-wordpress-and-drupal-with-sucuri.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+## Configure DNS
+From within your Sucuri Account:
+
+1. Click on **General Settings**.
+1. Copy the **Firewall IP Address**.
+
+From within your DNS host:
+
+<ol start="3"><li> Delete the existing <code>www</code> <strong>CNAME</strong> record that points to Pantheon.</li>
```
Comment (bmackinney) [inline source/docs/articles/sites/domains/defend-wordpress-and-drupal-with-sucuri.md]: @atjuch Do not all of the CNAME records for the site need to be added through Sucuri?

Comment (atjuch) [inline source/docs/articles/sites/domains/defend-wordpress-and-drupal-with-sucuri.md]: @bmackinney If there are several domains under the Live site you wouldn't be able to because Sucuri only supports one domain at a time. For example, if the user had 10 domains under the Live Pantheon "Domains/SSL" area, on their Sucuri account they would need to pay for 10 domains on their CloudProxy since the CloudProxy support one paid domain at a time.

Comment (bmackinney) [inline source/docs/articles/sites/domains/defend-wordpress-and-drupal-with-sucuri.md]: What about the CNAME record if only for `www.example.com`? I recall seeing that in the Sucuri instructions.

Comment (atjuch) [inline source/docs/articles/sites/domains/defend-wordpress-and-drupal-with-sucuri.md]: @bmackinney Yes, you are correct. Part of the clean-up that @rachelwhitton did might have removed some of the information regarding the CNAME along with a screenshot.

Here is what the CNAME settings should look like within Sucuri http://screencast.com/t/jbC3qT0z

Comment (rachelwhitton) [inline source/docs/articles/sites/domains/defend-wordpress-and-drupal-with-sucuri.md]: @atjuch @bmackinney the "CNAME" record within Sucuri is referred to as **Hosting IP Address** 

Instructions for how to add destinations (shown in @atjuch's screenshot above) is step 5 of the Configure DNS section.
